### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by lilaczhou** [vitoralbiero/keras_drml_region_layer#1]
Do you know how to code the multilabel sigmoid loss function defined in the paper in Keras?
![image](https://user-images.githubusercontent.com/45725450/59413122-83d63880-8df1-11e9-9046-030539270ab9.png)

**Comment by vitoralbiero** [comment]
In Keras you can do,
in the last layer: Dense(10, activation='sigmoid')
and loss = ['binary_crossentropy']

I have it implemented in https://github.com/vitoralbiero/drl_action_unit_detection

Check inside fexp/model/drl.py.

**Comment by vitoralbiero** [comment]
In Keras you can do,
in the last layer: Dense(10, activation='sigmoid')
and loss = ['binary_crossentropy']

I have it implemented in https://github.com/vitoralbiero/drl_action_unit_detection/blob/master/fexp/model/drl.py

**Comment by lilaczhou** [comment]
But the definend loss function is different from binary_crossentropy offered in Keras,we need to self-define,do you know how to implement it?

**Comment by vitoralbiero** [comment]
Actually, they are the same, this is the binary cross entropy loss in Keras: 
![image](https://user-images.githubusercontent.com/7631057/59525224-d07a4c00-8ea3-11e9-9f9b-a790f246de21.png)
